src/components/LoginForm/LoginForm.js

```diff
@@ -1,28 +1,53 @@
 import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
-import * as actions from '../../actions';
 
-import { Button, Form } from '../common';
+import * as actions from '../../actions';
+import { Button, Form, Toast } from '../common';
+import api from '../../apis/api'
 
 import styles from './LoginForm.module.css';
 
 class LoginForm extends Component {
 
+  state = { errorMesasge: null }
+
   loginDetails = { email: '', password: '', errorMessage: null }
 
   handleInput = (stateProp, value) => {
     this.setState({ [stateProp]: value })
   }
 
-  loginSubmit(state) {
-    console.log('loginSubmit()')
-    console.log(state)
-    // const { email, password } = this.state;
-    // this.props.changeAuth({ email, password }, (error) => {
-    //   this.setState({ errorMessage: error })
-    // })
+  loginSubmit = (state) => {
+    const { email, password } = state;
+    // console.log('this: ', this)
+    api.post('/auth/login', { email, password })
```

Comment:
Should this not be in an action, and then state updated in a reducer?  Its like subscribing to an event